cov.el

```diff
@@ -266,6 +264,7 @@ code's execution frequency"
 
 (defun cov-turn-on ()
   "Turn on cov-mode."
+  (cov-clear-overlays)
```

Comment:
Tests unveiled that, if you re-enable the mode the overlays are re-created. Clearing them beforehand solves this.